apps/vanilla/src/pages/index.html

```diff
@@ -1,16 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+		<link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./index.css" />
-    <title>Vite App</title>
+    <title>Book App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
+  <body class="page">
+		<header class="header">
+			<label class="font-style_logo">📚 Books app</label>
+		</header>
+    <main class="main">
+			<div class="filters col_3 col_s_12">
+				<label class="font-style_subtitle">filters</label>
+				<div class="filters-controls">
+					<input class="filters-control font-style_body" type="search" placeholder="e.g. “Refactoring”" name="search">
+					<select class="filters-control select font-style_body" required>
```

Comment:
```suggestion
					<select class="filters-control select font-style_body">
```

Most often, filters are not required, but if you need a required value, it is better to remove the option to select an empty value

Reply:
I found that using `required` option is the only way how I can change text color at select field for default value.